std/concurrency.d

```diff
@@ -407,9 +407,9 @@ public:
     assert(res == "Child responding");
 }
 
-//////////////////////////////////////////////////////////////////////////////
+// ////////////////////////////////////////////////////////////////////////////
```

Comment:
Just kill the row, keep it at `//`. It looked weird before it's way more weird now.